utils.py

```diff
@@ -24,9 +24,9 @@
 import time
 
 
-def dbg(fmt, *args):
+def dbg(*args, **kwargs):
```

Comment:
I'm not sure I understand the design of this wrapper. Maybe have impl be `print(*args, **kwargs, file=sys.stderr, flush=True)`? Then the api is same as print function.

Reply:
That's quite clever let me change it to that :)